skimage/filters/thresholding.py

```diff
@@ -910,7 +910,7 @@ def apply_hysteresis_threshold(image, low, high):
     .. [1] J. Canny. A computational approach to edge detection.
            IEEE Transactions on Pattern Analysis and Machine Intelligence.
            1986; vol. 8, pp.679-698.
-           DOI: 10.1109/TPAMI.1986.4767851
+           :DOI:` 10.1109/TPAMI.1986.4767851`
```

Comment:
extra space? does it matter?

Reply:
You're correct. It should be remove, I missed that one.